lib/sms.rb

```diff
@@ -0,0 +1,29 @@
+require 'twilio-ruby'
+require 'dotenv'
+
+class Sms
+  attr_reader :credentials
+
+  def initialize
+    @credentials = Dotenv.load
+    @client = twilio_client
+
+  end
+
+  def send(message, number)
+    @client.messages.create(
+     from: credentials["TWILIO_NUMBER"],
+     to: credentials["TWILIO_MOBILE"],
+     body: message
+     )
```

Comment:
Style/ClosingParenthesisIndentation: Indent `)` the same as the start of the line where `(` is.